### ISSUE TITLE
Tag Maker | Tag title is not getting updated from CSV

### ISSUE THREAD

**Issue by fechner** [Adobe-Consulting-Services/acs-aem-tools#219]
**Steps to reproduce:**
1. Import CSV file with tags
2. Update title of one tag and import CSV again

**Expected:**
Tag title is updated

**Actual:**
Tag title is not updated

**Tech Notes:**
It works fine on AEM 6.3 but the problem is on AEM 6.5 . This is because TagManager.createTag method in AEM 6.5 is not updating tag title. So for new AEM you need to update this title manually as it is in TagCreator (check setTitles method)

**Comment by davidjgonzalez** [comment]
Just to confirm, this works in ACS AEM Commons TagCreator, correct? TagCreator has ~replaced TagMaker since its in ACS Commons (which is OK to install/use on Production) where as ACS Tools is really meant for lower envs.

https://adobe-consulting-services.github.io/acs-aem-commons/features/mcp-tools/tag-creator/index.html

**Comment by fechner** [comment]
Yes, it works in Tag Creator but as you know Tag Creator use Excel file instead of CSV file. In our case it's little complicated to change file format. That's why we still want to use Tag Maker.

**Comment by davidjgonzalez** [comment]
@fechner gotcha - i assume youre exporting the CSV from some other system, rather than managing it in Excel to begin w?

I wonder how easy it would be to have TagCreator convert CSV -> Spreadsheet format inmemory and process it like that... 🤔
